File: challenges/3-area-trapezio/python/jvhk/area-trapezio.py
try:
    baseMaior = float(input("Digite o valor da base maior do trapézio: "))
except ValueError:
    baseMaior = float(input("O valor digitado não é um número, digite um número para a base maior do trapézio: "))

try:
    baseMenor = float(input("Digite o valor da base menor do trapézio: "))
except ValueError:
    baseMenor = float(input("O valor digitado não é um número, digite um número para a base menor do trapézio: "))

try:
    altura = float(input("Digite o valor da altura do trapézio: "))
except ValueError:
    altura = float(input("O valor digitado não é um número, digite um número para a altura do trapézio: "))

Area = (baseMaior + baseMenor) * altura / 2

print("A área do trapézio é: {:.1f}".format(Area))

File: challenges/3-area-trapezio/java/guilhermeaureliano/AreaTrapezio.java
import java.util.Scanner;

public class AreaTrapezio {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		
		System.out.print("Informe a base maior do trapézio: ");
		double baseMaior = input.nextDouble();
		
		System.out.print("Informe a base menor do trapézio: ");
		double baseMenor = input.nextDouble();
		
		System.out.print("E informe a altura do trapézio: ");
		double altura = input.nextDouble();
		
		double area = ((baseMaior + baseMenor) * altura) / 2;
		System.out.println("A área do trapézio é: " + area);
		
		input.close();
	}
}
